tests/ui/copies.rs

```diff
@@ -335,6 +335,17 @@ fn if_same_then_else() -> Result<&'static str, ()> {
         let foo = "";
         return Ok(&foo[0..]);
     }
+
+    // See #3559
```

Comment:
nit: I'd like this line to be slightly more descriptive, something like
````    // false positive if_same_then_else, let(x,y) vs let(y,x), see #3559````

Reply:
Fixed that. Thanks for feedback!